app/templates/components/ui-table/cell/cell-buttons.hbs

```diff
@@ -1,17 +1,20 @@
-<div class="ui vertical compact basic buttons">
-  {{#ui-popup tagName='a' click=(action moveToDetails record.identifier) content=(t 'View') class='ui icon button' position='left center'}}
+<div class="ui horizontal compact basic buttons">
```

Comment:
Do the following : 
- Take this code and paste it under the template which handles the Name column
- Delete this file.
- Make the button size responsive